src/Spectre.Tests/Models/UserModelTest.cs

```diff
@@ -0,0 +1,8 @@
+﻿using System.Web.Mvc;
```

Comment:
Those files (from *Spectre.Tests* project) were commited earlier, as a part of another task. Nevertheless, I will add those headers.